firebase-sessions/src/main/kotlin/com/google/firebase/sessions/FirebaseSessions.kt

```diff
@@ -139,7 +139,7 @@ internal constructor(
     }
 
     sessionCoordinator.attemptLoggingSessionEvent(
-      SessionEvents.startSession(firebaseApp, sessionDetails, sessionSettings)
+      SessionEvents.startSession(firebaseApp, sessionDetails, sessionSettings, subscribers)
```

Comment:
Is this pass by value of reference? Since we are passing this around for every event, we would be more recent. But still checking if there are any race conditions involved here.

Reply:
The `subscribers` map is a "read only" map that gets copied every time we call `FirebaseSessionsDependencies.getRegisteredSubscribers()`. So it won't change between here and the `startSession` unless we do it on purpose, we would have to cast it to a `MutableMap`.